### ISSUE TITLE
Double-clicking in the file explorer opens two files dues to Open Editors view expanding

### ISSUE THREAD

**Issue by realh** [microsoft/vscode#128508]
Issue Type: <b>Bug</b>

1. Have the "OPEN EDITORS" section of the explorer open, and above the explorer's main section.
2. Double-click one of the files in the main section, but delay the second click slightly.

The file opens on the first click and gets added to the OPEN EDITORS section. This moves everything below it down by the height of one line, so if that happens before the second click, you're clicking on a different file. Often, instead of opening the second file, it opens a text entry field near the bottom of the explorer's main section (not necessarily anywhere near the mouse cursor) as if I'm trying to create a new file at the top-level of my project.

VS Code version: Code 1.57.1 (507ce72a4466fbb27b715c3722558bb15afa9f48, 2021-06-17T13:26:56.255Z)
OS version: Linux x64 5.12.14-arch1-1
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6500U CPU @ 2.50GHz (4 x 1015)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 1, 1|
|Memory (System)|7.64GB (1.52GB free)|
|Process Argv|--no-sandbox --unity-launch --crash-reporter-id f284bc02-8148-43a2-b910-cc5852739e93|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|gnome|
|XDG_CURRENT_DESKTOP|GNOME|
|XDG_SESSION_DESKTOP|gnome|
|XDG_SESSION_TYPE|wayland|
</details><details><summary>Extensions (22)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-styled-jsx|bla|2.0.0
vscode-modelines|chr|2.0.5
vscode-eslint|dba|2.1.23
vscode-html-css|ecm|1.10.2
tslint|eg2|1.0.47
auto-close-tag|for|0.5.11
dotenv|mik|1.0.1
vscode-language-pack-en-GB|MS-|1.54.1
python|ms-|2021.6.944021595
vscode-pylance|ms-|2021.7.2
jupyter|ms-|2021.6.999662501
remote-containers|ms-|0.183.0
remote-ssh|ms-|0.65.7
remote-ssh-edit|ms-|0.65.7
remote-wsl|ms-|0.56.5
vscode-remote-extensionpack|ms-|0.21.0
cpptools|ms-|1.5.1
lit-plugin|run|1.2.1
sourcekit-lsp|unp|0.0.1
vim|vsc|1.21.5
viml|Xad|1.0.1
html-css-class-completion|Zig|1.20.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551cf:30311713
vspre833:30321513
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscus158:30321503
pythonvsuse255:30335481
vscod805:30301674
pythonvspyt200:30331937
vscextlangct:30333562
binariesv615:30325510
bridge0708:30335490

```

</details>

**Comment by isidorn** [comment]
This is a known limitation. However I could not find the duplicate so let's keep this one open.

**Comment by realh** [comment]
I did subsequently find the issue that this duplicates, but it had been closed by a bot: #25911.

**Comment by bpasero** [comment]
The limit is at `250ms` currently but in theory can be changed to a higher value:

https://github.com/microsoft/vscode/blob/ae86e4c50532d12309dbc1bb7c9579b962fbe42e/src/vs/workbench/contrib/files/browser/explorerViewlet.ts#L197-L229
